### ISSUE TITLE
Is it possible to plot the Quantization Error (and the Topographic Error) trend for each epoch?

### ISSUE THREAD

**Issue by f-carella** [JustGlowing/minisom#200]
Dear Admins, 

I am working with this tool with settings random_order=True and use_epochs=True. I would like to plot the Quantization Error and the Topographic Error trend as a function of the number of epochs. Meaning QE vs Epochs and Te vs Epochs. Is it possible to retrieve this information from the trained model? If not, how do you retrieve this information?

**Comment by JustGlowing** [comment]
hi @f-carella , you can't do that from a model that you've already trained but you can do that for models that you will train. The last snippet of code in this notebook shows how to do it: https://github.com/JustGlowing/minisom/blob/master/examples/BasicUsage.ipynb

**Comment by f-carella** [comment]
Hi, thanks. Yes, I had a look at that, but is not that a different case? Because that example uses the number of iteration and not the number of epochs. The issue is that per epoch, you have that sigma and learning rate are fixed, and they just decrease after one epoch (after you see all the data), whereas in the example, sigma decreases after each iteration, right?

The main reason why I am doing this is because I would like to have a sort of early stopping on the quantization error during the training.

**Comment by JustGlowing** [comment]
You're right, that examples shows how to plot the error at each iteration. You'll also notice that the example shows how to define an iteration and the parameters, you need to adapt that to your definition of epoch and compute the error accordingly.
